tests/meta-mender-beaglebone-ci/README

```diff
@@ -26,3 +26,41 @@ General
 
 This is a Mender QA specific layer. The layer contains the necessary bits and
 pieces for use with Mender CI pipeline.
+
+Setup
+=====
+
+* S2 switch is pressed when powering on the device, the board should boot using
+  MMC1 device, this will be visible in U-Boot banner:
+
+      U-Boot SPL 2017.05 (Aug 07 2017 - 08:55:56)
+      Trying to boot from MMC1
+
+
+Test host
+---------
+
+* Use mender-image-test-host to build a test host image
+
+* Test host image is running from eMMC (/dev/mmcblk1 aka `mmc dev 1` under U-Boot)
+
+* When built, just dd the image onto the card, boot, once in the running system do:
+
+     dd if=/dev/mmcblk0 of=/dev/mmcblk1 bs=1M
+
+* Set MACHINE = "beaglebone-mender" device when building for test host.
```

Comment:
Why is it not enough to use the CI distro?

Reply:
We're not setting any machine specific configuration for BBB yet, but it's already being done for Toradex. I've added `beaglebone-mender` just to have a placeholder for any such settings. For instance, should we change the testing to use internal eMMC rather than the SD card (which probably makes more sense for real life application), the related settings would go to `beaglebone-mender.conf`.